Bottom: 0b5f0a604099929392248ab1e63c561b670093b1
Top:    ea132c2261ea7ea486f8d207e7aff4187cc2e743
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-05-05 18:48:17 +0100

Export multi-body parts, not a bag of loose parts due to our incomplete step assy emission code.


---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 76ab121..075f811 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -225,8 +225,8 @@ step_do_export (HID_Attr_Val * options)
   } while ((piece = piece->f) != board_outline);
   poly_Free (&board_outline);
 
-//  object3d_list_export_to_step_part (board_outline_list, temp_pcb_filename);
-  object3d_list_export_to_step_assy (board_outline_list, temp_pcb_filename);
+  object3d_list_export_to_step_part (board_outline_list, temp_pcb_filename);
+//  object3d_list_export_to_step_assy (board_outline_list, temp_pcb_filename);
   g_list_free_full (board_outline_list, (GDestroyNotify)destroy_object3d);
 
   if (1) {
